source/declarations.tex

```diff
@@ -2507,15 +2510,15 @@
 the target scope of the \grammarterm{declarator} is that scope.
 \end{itemize}
 \item
-Otherwise, the terminal name of $E$ is not looked up.
-The declaration's target scope is the innermost enclosing namespace scope;
-if the declaration is contained by a block scope,
+Otherwise,
+the declaration's target scope is the innermost enclosing namespace scope.
+If the declaration is contained by a block scope,
 the declaration shall correspond to a declaration
 that inhabits the innermost block scope.
 \end{itemize}
 
 \pnum
-Otherwise:
+If the declaration is not a friend declaration:
```

Comment:
As this is exhaustive together with the previous paragraph, I'd prefer to use one of our formulations that indicates that: perhaps
> Otherwise, the declaration is not a friend declaration:

or
> Otherwise (for a non-friend declaration):